### DISCUSSION TITLE
Instance being marked as deceptive by google

### DISCUSSION THREAD

**Discussion by ggilestro** [mastodon/mastodon#19853] [Q&A]
My self-hosted docker based Mastodon instance has been marked as deceptive by Google. The same happened to the instance of a friend, hosted by masto.host.

Any idea why this may happen? The google report is not informative at all. I have looked all the JS being served and they are not modified in any way.

![image](https://user-images.githubusercontent.com/696901/200172039-769aee5a-9f18-402f-8e22-e2033adc6b2d.png)

For good measure I have restarted the instances and donwloaded new images, using the opportunity to upgrade to 4.0.0rc2 but I do not quite understand whether really there is a vulnerability of whether this is an attempt at boycotting Mastodon in this important moment.

**Comment by matthewford** [comment]
You can request a review, sometimes they give more information.

**Reply by ghost** [reply]
You can try but Google doesn't really look at agpl licensed code to audit

**Reply by matthewford** [reply]
they look at the html/css/js not the backend code

**Comment by ghost** [comment]
> My self-hosted docker based Mastodon instance has been marked as deceptive by Google. The same happened to the instance of a friend, hosted by masto.host.
> 
> Any idea why this may happen? The google report is not informative at all. I have looked all the JS being served and they are not modified in any way.
> 
> ![image](https://user-images.githubusercontent.com/696901/200172039-769aee5a-9f18-402f-8e22-e2033adc6b2d.png)
> 
> For good measure I have restarted the instances and donwloaded new images, using the opportunity to upgrade to 4.0.0rc2 but I do not quite understand whether really there is a vulnerability of whether this is an attempt at boycotting Mastodon in this important moment.

Mastodon is not a central website like Twitter, but a free software tool for micro blogging. If your source code checks out in compliance with the agpl I see no deception

**Comment by ggilestro** [comment]
It turns out Google will do this for mastodon instances whose DNS is not registered on the Google search console. After registration of ownership they remove the red flag within 24h. This is of course very frustrating.

**Reply by ghost** [reply]
It's like getting a verification blue check mark from Twitter, a hassle
